package.json

```diff
@@ -47,6 +48,7 @@
     "@storybook/addon-actions": "6.2.9",
     "@storybook/addon-docs": "6.2.9",
     "@storybook/addon-knobs": "6.2.9",
+    "@storybook/addon-storyshots": "^6.3.4",
```

Comment:
storybook/addon-storyshots version should match other storybook version numbers (6.2.9)